front/lib/connector_providers.ts

```diff
@@ -37,7 +39,8 @@ export const CONNECTOR_CONFIGURATIONS: Record<
     isBuilt: true,
     logoPath: "/static/google_drive_32x32.png",
     description:
-      "Authorize granular access to your company's Google Drive, by drives and folders. Supported files include GDocs, GSlides, and .txt files.",
+      "Authorize granular access to your company's Google Drive, by drives and folders. Supported files include GDocs, GSlides, and .txt files. Email us for .pdf indexation.",
+    limitations: "Files with more than 750KB of extracted text are ignored.",
```

Comment:
Now the we have more room, I would shortly explain why and be clearer on "extracted" VS "file size".

"Dust will read all files including large PDFs but will ignore them when extracted text content is bigger than 750Kb (~XXX pages of texts)"